### ISSUE TITLE
opentracing-redis-spring doesn't support ReactiveRedisConnectionFactory

### ISSUE THREAD

**Issue by chanjarster** [opentracing-contrib/java-redis-client#11]
I try to use `opentracing-redis-spring` in an spring boot application like this:

```java
@Component
public class RedisConnectionFactoryBeanPostProcessor implements BeanPostProcessor {

  private Tracer tracer;

  public RedisConnectionFactoryBeanPostProcessor(Tracer tracer) {
    this.tracer = tracer;
  }

  @Override
  public Object postProcessAfterInitialization(Object bean, String beanName) throws BeansException {
    if (bean instanceof RedisConnectionFactory) {
      return new TracingRedisConnectionFactory((RedisConnectionFactory) bean, false, tracer);
    }
    return bean;
  }
}
```

And I got this error:

```
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'reactiveRedisTemplate' defined in class path resource [org/springframework/boot/autoconfigure/data/redis/RedisReactiveAutoConfiguration.class]: Unsatisfied dependency expressed through method 'reactiveRedisTemplate' parameter 0; nested exception is org.springframework.beans.factory.BeanNotOfRequiredTypeException: Bean named 'redisConnectionFactory' is expected to be of type 'org.springframework.data.redis.connection.ReactiveRedisConnectionFactory' but was actually of type 'io.opentracing.contrib.redis.spring.connection.TracingRedisConnectionFactory'
Caused by: org.springframework.beans.factory.BeanNotOfRequiredTypeException: Bean named 'redisConnectionFactory' is expected to be of type 'org.springframework.data.redis.connection.ReactiveRedisConnectionFactory' but was actually of type 'io.opentracing.contrib.redis.spring.connection.TracingRedisConnectionFactory'
```

I found that `TracingRedisConnectionFactory` only implements `RedisConnectionFactory` while `spring-boot-starter-data-redis` auto configures an `LettuceConnectionFactory` by default which also implements `ReactiveRedisConnectionFactory` and that's why the error appears.

**Comment by malafeev** [comment]
@chanjarster I will fix it

**Comment by malafeev** [comment]
fixed in version 0.0.8.
will be available in maven central during next 24 hours.
